src/main/java/mekanism/common/item/ItemAtomicDisassembler.java

```diff
@@ -368,10 +366,11 @@ public Finder(EntityPlayer p, ItemStack s, Coord4D loc, RayTraceResult traceResu
             rayTraceResult = traceResult;
             isWood = OreDictCache.getOreDictName(stack).contains("logWood");
             maxRange = range;
+            maxCount = MekanismConfig.current().general.DISASSEMBLER_COUNT.val() - 1;
```

Comment:
With the -1 doesn't this change the amount that it actually can do? Or is this fixing that it was previously doing 129 before (haven't looked too closely at the code yet)

Reply:
it fixes mining one more than set in the config.